docs/admin/runai-setup/self-hosted/k8s/backend.md

```diff
@@ -34,57 +34,67 @@ Run the helm command below:
         --set global.domain=<DOMAIN>  # (2)
         -n runai-backend -f custom-env.yaml  # (3)
     ```
-       
+
     1. Replace `<VERSION>` with the Run:ai control plane version.
     2. Domain name described [here](prerequisites.md#domain-name). 
     3. `custom-env.yaml` should have been created by the _prepare installation_ script in the previous section. 
 
 !!! Tip
-    Use the  `--dry-run` flag to gain an understanding of what is being installed before the actual installation. 
-
-## (Air-gapped only) Local Certificate Authority
-
-Perform the instructions for [local certificate authority](../../config/org-cert.md). 
```

Comment:
@jasonnovichRunAI this was not a part of my PR and was somehow deleted... need to bring it back